src/main/java/org/gridsuite/actions/server/ContingencyListService.java

```diff
@@ -44,8 +43,6 @@
 @Service
 public class ContingencyListService {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(ContingencyListService.class);
```

Comment:
why are the loggers removed ?

Reply:
I think it is useless the log in debug the entrance of basic functions. Furthermore, it has an effect on test coverage because of the IF.